paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/AppearanceStore.kt

```diff
@@ -7,10 +7,8 @@ import com.stripe.android.paymentsheet.PaymentSheet
 
 internal object AppearanceStore {
     var state by mutableStateOf(PaymentSheet.Appearance())
-    var forceDarkMode by mutableStateOf(false)
```

Comment:
Can remove this workaround since we’re no longer relying on an activity-scoped dark mode.